flutter v9: update snippets (#13496)
getsentry/sentry-docs: platform-includes/enriching-events/attach-screenshots/dart.flutter.mdx

### Customize Screenshot Capturing

By default, Flutter limits screenshot captures to once every 2 seconds to minimize performance impact. While this debounce interval cannot be changed, you can customize capture behavior by implementing the `beforeCaptureScreenshot` callback in your `SentryFlutterOptions`.

This callback gives you fine-grained control over screenshot captures based on event and hint data, allowing you to implement conditional logic.

Return `true` to capture the screenshot, or `false` to skip it.

<Alert>

> The `debounce` flag is `true` if the debounce is active and would prevent the screenshot from being captured.

Reviewer: ```suggestion
The `debounce` flag is set to `true` when debounce is active, meaning it will block the screenshot from being taken..
```

> If you want to capture screenshots regardless of the debounce, you ignore the `debounce` flag.

Reviewer: ```suggestion
If you want to capture screenshots regardless of the debounce, you can ignore the `debounce` flag.
```

</Alert>

```dart {tabTitle: 'Respect debounce'} {2-9}
await SentryFlutter.init((options) {
  options.beforeCaptureScreenshot = (event, hint, shouldDebounce) async {
    // If shouldDebounce is active, skip capturing
    if (shouldDebounce) {
      return false;
    }
    // Capture screenshot if it's a fatal event
    return event.level == SentryLevel.fatal;
  };
});
```

```dart {tabTitle: 'Ignore debounce'} {2-4}
await SentryFlutter.init((options) {
  options.beforeCaptureScreenshot = (event, hint, shouldDebounce) async {
    // Capture screenshot if it's a fatal event regardless of debounce
    return event.level == SentryLevel.fatal;
  };
});
```

## Redact Screenshots via `masking`

> The masking feature is enabled by default for screenshots.

Reviewer: ```suggestion
The masking feature is enabled by default for screenshots, but can be disabled or configured by adjusting the `options.experimental.privacy` option.
```

> If you want to disable or configure the masking feature, you can do so by adjusting the `options.experimental.privacy` option.

Reviewer: ```suggestion
```
